src/dxtbx/format/FormatHDF5EigerNearlyNexus.py

```diff
@@ -359,10 +359,12 @@ def _start(self):
             detector, self._beam_factory.model, shape=self._raw_data.shape()
         ).model
 
-        # Override the minimum trusted value - for Eiger should be -1
+        # Override the minimum trusted value - for Eiger should be 0.
+        # This needs a careful check in the context of testing the
+        # trust-the-trusted_range branch
```

Comment:
Is this comment asking for someone to do this check?

Reply:
Yes, I'm not actually sure if `FormatHDF5EigerNearlyNexus` is used. Maybe it is for old files only? I haven't done any tests of real datasets outside of the test suite, so welcome any efforts here. If things are good then comment spam like this can be removed.

Reply:
oh, the other thing is that I was not really sure why the code here is doing this. It overrides the trusted range written in the file for some reason. I guess that is something to do with files that were written incorrectly? Anyway, I don't know the full history here.

Reply:
Ok ya I bet the underload just wasn't specified in the file.